frontend/src/components/AddElection.tsx

```diff
@@ -1,17 +1,22 @@
 import { useState } from "react"
 import React from 'react'
 import { useNavigate } from "react-router"
+import {Election} from './../../../domain_model/Election'
```

Comment:
nit

``./../../../domain_model/Election`` is equivalent to ``../../../domain_model/Election`` right?